packages/driver/src/cy/commands/angular.ts

```diff
@@ -91,7 +89,7 @@ export default (Commands, Cypress, cy, state) => {
   }
 
   Commands.addAll({
-    ng (type, selector, options = {}) {
+    ng (type, selector, options: any = {}) {
```

Comment:
It might be worth adding an explicit type for options if we are reading properties off of it. Or is this represented in the [types](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts) already?